relay-server/src/endpoints/common.rs

```diff
@@ -325,11 +325,22 @@ pub async fn handle_envelope(
     state: &ServiceState,
     envelope: Box<Envelope>,
 ) -> Result<Option<EventId>, BadStoreRequest> {
+    let client_name = envelope.meta().client_name();
     for item in envelope.items() {
         metric!(
             histogram(RelayHistograms::EnvelopeItemSize) = item.payload().len() as u64,
             item_type = item.ty().name()
         );
+        metric!(
+            counter(RelayCounters::EnvelopeItems) += 1,
```

Comment:
Should we only count in the loop and emit once after, to minimize the work in `statsdproxy`?

Reply:
We also need the `item_type` in the metric so I think it wouldn't really change much

Reply:
Ah right fair enough 👍